### ISSUE TITLE
Render Settings & View enhancements

### ISSUE THREAD

**Issue by hexabits** [niftools/nifskope#25]
#### Expose Addt'l Render Settings

Many more render settings can be exposed to the user.  Just off the top of my head:
- [ ] Vertex Size
- [ ] Wire width
- [ ] Normal/(Bi)Tangent handle size
- [x] FOV (field of view) for perspective camera
- [x] Camera Move & Rotation speeds

I have already done two of these in misc. builds.  I also just discovered that FPS is strangely set to 25, which must be due to European developers.  This FPS limit should either be configurable or dependent on refresh rate.
#### Usability Enhancements in Render View

Also, not just limited to Render > Settings, there are many other things that can be toggled by the user to ease NIF editing:
- [x] Texture toggle<sup>[1]</sup>  | Shortcut: Alt+T  
- [x] Vertex Color toggle  | Shortcut: Alt+V  
- [x] Shaded / Flat Lighting toggles<sup>[2]</sup>
- [ ] Force Double Sided toggle  
- [ ] (Solid / Transparent / Wireframe) toggle <sup>[3]</sup>
- [ ] Mesh (block) visibility <sup>[4]</sup>
- [ ] (Normal / Alpha / Specular / etc) as Diffuse toggles <sup>[5]</sup>

[1]: In addition to or in lieu of setting in Render > Settings
[2]: For visualizing diffuse maps and vertex colors more clearly
[3]: For visualizing mesh interiors and collision coverage
[4]: For visualizing blocks hidden by others.  Note:  This currently requires at least one   mesh to be visible or the view stops rendering.  This can be fixed.
[5]: Since the shader pipeline appears broken to me (it's always fixed function) at least temporarily allow user to view normal maps, etc. in place of diffuse map.

**Comment by neomonkeus** [comment]
From a usability p.o.v its very difficult for initial users to find Render -> settings especially problematic when setting up texture pathing. 

A possible proposal for reworking the Render -> settings menu, perhaps to function like the Spells menu, where the tabs of the old "Render -> Settings" window are replaced with sub-dropdowns.

Non-render specific settings, like those found in the general & export tab should be put into a separate Header Menu.

The same would apply for all the "view" related items in the Render dropdown, like Top, Front, Side, Save View. 

I can open a separate issues for each if you would prefer, and am open to any direction for ui improvement but the above would be specifically about getting context related setting in the correct place.

**Comment by hexabits** [comment]
That's more related to Issue #13, no?  I made this separate from that because this is more simply about fleshing out more options for Settings > Rendering tab.  And some are about more options related to the Render View that are menu options themselves, like all the toggles.

We can open one large issue for UI restructuring.  I just need to get around to organizing an outline for that.  I just have a lot of different thoughts floating around at the moment.

**Comment by neomonkeus** [comment]
Yeah that issue would be appropriate for moving things out. A summary issue for general UI refactoring much in the same way you have the task list here would work well that.
